easybuild/easyconfigs/p/PyCUDA/PyCUDA-2016.1.2-intel-2016b-Python-2.7.12.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'PythonPackage'
+
+name = 'PyCUDA'
+version = '2016.1.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://mathema.tician.de/software/pycuda'
+description = """PyCUDA lets you access Nvidia’s CUDA parallel computation API from Python."""
+
+toolchain = {'name': 'intel', 'version': '2016b'}
+
+source_urls = [PYPI_LOWER_SOURCE]
+sources = [SOURCELOWER_TAR_GZ]
```

Comment:
@wpoely86 please keep an eye on the presence of checksums, we should start forcing that in Travis soon (cfr. #5005)